Header header
geometry_msgs/Vector3 shin_acc
geometry_msgs/Vector3 shin_gyr
geometry_msgs/Vector3 shin_mag
geometry_msgs/Vector3 shin_ref

